{"name": "test_", "status": "broken", "statusDetails": {"message": "NameError: name 'url' is not defined", "trace": "self = <taskOne.TestEditionControlSystem object at 0x00000116B6425860>\n\n    @allure.feature(\"获取所有版本数据\")\n    def test_(self):\n        path = \"getAllEditionData\"\n        \"\"\"登录接口\"\"\"\n        logging.info(\"用例 test_01 执行完成。。\")\n>       print(url + path)\nE       NameError: name 'url' is not defined\n\ntaskOne.py:31: NameError"}, "start": 1627570210995, "stop": 1627570210996, "uuid": "5d41e03f-83d4-4276-960b-babda21b8abc", "historyId": "a6602bdec1e6ad61fd2e2add1561cc53", "testCaseId": "260a32e0b710c9c32f93c76acdc6ff63", "fullName": "taskOne.TestEditionControlSystem#test_", "labels": [{"name": "feature", "value": "获取所有版本数据"}, {"name": "suite", "value": "taskOne"}, {"name": "subSuite", "value": "TestEditionControlSystem"}, {"name": "host", "value": "LAPTOP-CDMV833B"}, {"name": "thread", "value": "9044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "taskOne"}]}